.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,33 @@
+name: Dependabot auto-merge
+
+on:
+  pull_request_target:
+  push:
+  workflow_dispatch:
```

Comment:
I think you're including all these triggers just for exploration purposes, is that right?

Reply:
yep, trying to cover all the possibilities 👍